code/obj/item/pens_writing_etc.dm

```diff
@@ -318,6 +318,21 @@
 				user.visible_message("<span class='notice'><b>\"Something\" special happens to [src]!</b></span>")
 				JOB_XP(user, "Clown", 1)
 
+		robot
+			desc = "Don't shove it up your nose, no matter how good of an idea that may seem to you. Wait, do you even have a nose? Maybe something else will happen if you try to stick it there."
+
+			attack(mob/M as mob, mob/user as mob, def_zone)
+				if (M == user)
+					src.color = random_color()
+					src.font_color = src.color
+					src.color_name = hex2color_name(src.color)
+					src.name = "[src.color_name] crayon"
+					user.visible_message("<span class='notice'><b>\"Something\" special happens to [src]!</b></span>")
+					JOB_XP(user, "Clown", 1)
```

Comment:
Can we drop the infinite source of "click for clown XP" from this?

Reply:
I'm assuming it should also be removed from the other version of the crayon, then?